### ISSUE TITLE
Numpy for Python 3.6 only built against MKL

### ISSUE THREAD

**Issue by javiljoen** [ContinuumIO/anaconda-issues#1321]
There is no "nomkl" build for Numpy with Python 3.6 in the default channel. Any chance it could be added (along with Scipy and Statsmodels versions that depend on it)?

```
> conda search --spec numpy=1.11.2
Fetching package metadata .......
numpy                        1.11.2                   py27_0  defaults        
                             1.11.2             py27_nomkl_0  defaults        [nomkl]
                             1.11.2                   py34_0  defaults        
                             1.11.2             py34_nomkl_0  defaults        [nomkl]
                             1.11.2                   py35_0  defaults        
                          .  1.11.2             py35_nomkl_0  defaults        [nomkl]
                             1.11.2                   py36_0  defaults        
```

**Comment by ilanschnell** [comment]
I know.  Not all Python 3.6 packages are available yet (currently about 80% of the Python packages we support).  The plan is to have all Python packages (the ones which support Python 3) available by the time we release Anaconda 4.3, which will be at the end of January.  Also, the Anaconda3 installers will be Python 3.6 based.

**Comment by javiljoen** [comment]
Thanks for the info!
